test/e2e/utils/utils.go

```diff
@@ -40,6 +40,16 @@ const (
 	poll = time.Second
 )
 
+// GetFormattedErrorAndLog returns the error and the log of the container in string format
+func GetFormattedErrorAndLog(errPodNotSuccess error, ctx context.Context, f *framework.Framework, podName, containerName string) string {
+	log, err := e2epod.GetPodLogs(ctx, f.ClientSet, f.Namespace.Name, podName, containerName)
+	if err != nil {
+		framework.Logf("unable to get log from pod: %v", err)
+	}
+	return fmt.Sprintf("unable to finish pod: %v\n", errPodNotSuccess) +
```

Comment:
what Gomega prints when the assertion fails? I feel this is redundant.

Reply:
so you mean, we do not need errPodNotSuccess at all here, right?

Reply:
Then, let me just make it print logs, not error.
From the name you suggested earlier, I thought.. we still print errors inside the function.

Let me change it to `GetPodLogs`